test/SIL/bitwise_copyable.swift

```diff
@@ -0,0 +1,59 @@
+// RUN: %target-swift-frontend                         \
+// RUN:     %s                                         \
+// RUN:     -emit-sil                                  \
+// RUN:     -enable-experimental-feature ValueGenerics \
+// RUN:     -enable-builtin-module \
+// RUN:     -disable-availability-checking \
+// RUN:     -O
+
+// REQUIRES: swift_feature_ValueGenerics
+
+// Force verification of TypeLowering's isTrivial.
```

Comment:
Let's use the preexisting test/SILGen/bitwise_copyable.swift for this.

Reply:
This crash actually happens _after_ SILGen in the optimizer.

Reply:
Okay.